library/src/main/scala/scala/scalajs/runtime/RuntimeLong.scala

```diff
@@ -576,8 +576,9 @@ final class RuntimeLong(val lo: Int, val hi: Int)
     else js.Tuple4(quotLo, quotHi, remLo, remHi)
   }
 
-  // Support for intrinsics
+  // TODO Remove these. They were support for intrinsics before 0.6.6.
 
+  @deprecated("Use java.lang.Long.toBinaryString instead.", "0.6.6")
```

Comment:
Well, last time I explained, the reason was: an 0.6.5 sbt plugin could link an 0.6.6 library, and would therefore expect those to be there. However, since 0.6.6 breaks forward binary compatibility, that cannot happen. So in theory, we could remove them. In practice, we would have to add exceptions in MiMa for the `library`, and I'd rather avoid that if we can avoid it. If there is no good reason to break the binary compatibility of the library, we shouldn't, I believe (even if it's in `runtime`).